app/main/views/suppliers.py

```diff
@@ -87,6 +89,17 @@ def view_signed_agreement(supplier_id, framework_slug):
     if not supplier_framework.get('agreementReturned'):
         abort(404)
 
+    # build an OrderedDict of applied-for lotSlug against lotName, ordered by lotSlug
+    lot_slugs_names = OrderedDict(sorted(
```

Comment:
Yep this will give the right order for lots, namely:
Lot 1: Infrastructure as a Service (IaaS)
Lot 2: Platform as a Service (PaaS)
Lot 3: Software as a Service (SaaS)
Lot 4: Specialist Cloud Services (SCS)

Not sure if this is a lucky coincidence or if the lots were just numbered in alphabetical order by CCS in the first place.